RealEstateAgency.Generator/Program.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.Extensions.Options;
+using RealEstateAgency.Generator.Services;
+using RealEstateAgency.ServiceDefaults; 
+
+var builder = Host.CreateApplicationBuilder(args);
+
+builder.AddServiceDefaults();
+
+var natsConnectionString = "nats://localhost:4222";
```

Comment:
Что же вы мне раньше не написали я бы вам объяснил.
```suggestion
var natsConnectionString = builder.Configuration.GetConnectionString("nats");
```
<img width="1785" height="521" alt="image" src="https://github.com/user-attachments/assets/49d2db7f-2845-4f57-b077-5dff4614a70f" />